CHANGELOG.md

```diff
@@ -23,6 +23,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   ([#2750](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/2750))
 - `opentelemetry-instrumentation-django` Fix regression - `http.target` re-added back to old semconv duration metrics
   ([#2746](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/2746))
+- `opentelemetry-instrumentation-asgi` do not set url.full attribute for server spans
```

Comment:
```suggestion
- `opentelemetry-instrumentation-asgi` do not set `url.full` attribute for server spans
```

Reply:
Thanks for your feedback @lzchen. The change is included in the new commit.